README.md

```diff
@@ -79,7 +79,7 @@ DIST_OUTPUT_BUCKET=amcufa-$DATETIME-dist
 TEMPLATE_OUTPUT_BUCKET=amcufa-$DATETIME
 aws s3 mb s3://$DIST_OUTPUT_BUCKET-$REGION --region $REGION
 aws s3 mb s3://$TEMPLATE_OUTPUT_BUCKET --region $REGION
-./build-s3-dist.sh --template-bucket ${TEMPLATE_OUTPUT_BUCKET} --code-bucket ${DIST_OUTPUT_BUCKET} --solution-name amcufa --version ${VERSION} --region ${REGION} --profile ${PROFILE} | tee >( grep TEMPLATE >template )
+./deployment/build-s3-dist.sh --template-bucket ${TEMPLATE_OUTPUT_BUCKET} --code-bucket ${DIST_OUTPUT_BUCKET} --solution-name amcufa --version ${VERSION} --region ${REGION} --profile ${PROFILE} | tee >( grep TEMPLATE >template )
```

Comment:
Line 76 takes you into the deployment directory.

Reply:
The `msg "$msg"` ?
This did not work for me in my tests.

Reply:
I don't understand. What error are you seeing and what command did you run to get that error? And is your PR resolving that error or is this a new issue?

Reply:
I think this PULL request is obsolete. Sorry about that.

I made a mistake in step
`cd amazon-marketing-cloud-uploader-from-aws/deployment`
and just jumped to
`cd amazon-marketing-cloud-uploader-from-aws`
which explains the behavior I have been seeing.

Will run another test on this repository next week.